handshake.md

```diff
@@ -496,6 +502,13 @@ Reading a Cable Wire Protocol message MUST follow these steps:
 3. The resulting bytes `plaintext` may then be parsed as a Cable Wire Protocol
    message.
 
+### 5.5 End of stream
+After a complete Cable Wire Protocol message is sent, regardless of whether
+it consists of one or several fragments, the sender MUST write a message of
+`length=0`. This final message serves as an end of stream marker for the
```

Comment:
I don't believe this is true. A host may send *many* Cable Wire Protocol messages before choosing to terminate the stream.

I think this first sentence could instead read "When a host has decided to terminate the streaming exchange of messages, the host MUST send a message of length zero. A host receiving a message of length zero MUST respond with its own message of length zero. When a host has both send and received a length-zero message, it is safe to perform any underlying disconnect logic for the underlying transport."